ttt/constants.py

```diff
@@ -3,22 +3,28 @@
 PLAYER_2_NAME = 'Player 2'
 PLAYER_2_MARKER = 'X'
 
-HUMAN_PLAYER_STRING = 'human'
-SIMPLE_COMPUTER_STRING = 'simpleComputer'
-SMART_COMPUTER_STRING = 'smartComputer'
+HUMAN_PLAYER_STRING = 'HumanPlayer'
```

Comment:
Could enums be used instead of plain strings/dicts here? Or would that it make it more complicated?